### PR TITLE
Remove listContainer border while it's loading

### PR DESCRIPTION
**Pull request by junedomingo**
a

### CODE REVIEW

File: src/search/screens/search.screen.js [hunk 1]
```diff
@@ -227,7 +230,13 @@ class Search extends Component {
         {searchStart &&
           !noUsersFound &&
           !noReposFound &&
-          <View style={styles.listContainer}>
+          <View
+            style={[
+              styles.listContainer,
+              (isPendingSearchUsers || isPendingSearchRepos) &&
```
Comment (alejandronanez) [inline src/search/screens/search.screen.js]: I like this. Maybe you can do `const isLoading = isPendingSearchUsers || isPendingSearchRepos` and later add that constant in here (it's a little more explicit)

Comment (alejandronanez) [inline src/search/screens/search.screen.js]: What do you think @junedomingo ?

**Comment by alejandronanez** [comment]
Let us know @junedomingo if you think that change is necessary and if not, let us know so we can merge your changes!

File: src/search/screens/search.screen.js (hunk 1 above)
Comment (junedomingo) [inline src/search/screens/search.screen.js]: That works for me :-)

**Comment by junedomingo** [comment]
I also assign noResult to new constant for consistency, what do you think?

**Comment by andrewda** [comment]
Very clean! Looks good to me!! 👍

**Pull request merged**